librz/core/cmd_descs/cmd_meta.yaml

```diff
@@ -0,0 +1,407 @@
+            optional: true
+      - name: Cd.
+        cname: meta_data_at
+        summary: Show the data mark at the current offset
+        args: []
+      - name: Cd-
+        cname: meta_data_remove
+        summary: Remove the data mark from the current address
+        args: []
+      - name: Cd-*
+        cname: meta_data_remove_all
+        summary: Remove all data marks
+        args: []
+  - name: Ch
+    summary: Manage the "hidden" mark metainformation
+    subcommands:
+      - name: Ch
+        cname: meta_hidden
+        summary: Set the "hidden" mark to the current address / List all "hidden" marks
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: Ch-
+        cname: meta_hidden_remove
+        summary: Remove the "hidden" mark from the current address
+        args: []
+      - name: Ch-*
+        cname: meta_hidden_remove_all
+        summary: Remove all "hidden" marks
+        args: []
+  - name: Cm
+    summary: Manage the "magic" mark metainformation
+    subcommands:
+      - name: Cm
+        cname: meta_magic
+        summary: Set the magic to the current address / List all magic marks
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+          - name: format
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: Cm-
+        cname: meta_magic_remove
+        summary: Remove the magic mark from the current address
+        args: []
+      - name: Cm-*
+        cname: meta_magic_remove_all
+        summary: Remove all magic marks
+        args: []
+  - name: Cr
+    summary: Manage the "mun" mark metainformation
+    subcommands:
+      - name: Cr
+        cname: meta_run
+        summary: Set the magic to the current address / List all magic marks
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+        args:
+          - name: command
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: Cr-
+        cname: meta_run_remove
+        summary: Remove the "run" mark from the current address
+        args: []
+      - name: Cr-*
+        cname: meta_run_remove_all
+        summary: Remove all "run" marks
+        args: []
+
+  - name: Cs
+    summary: Manipulate string metainformation
+    subcommands:
+      - name: Cs
+        cname: meta_string
+        summary: Add string
+        args:
+          - name: size
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: Csl
```

Comment:
The `l` for "list" was added only for `Cs`, but not for `Cd` and all the others... I think we should either do it for all kind of meta commands or for none.

So i have two proposals:
1) we add `<cmd>l` for all the missing ones
2) we make `Cs <size>` instead of `Cs [size]` and we add `Csz` to have a string with an "automatic size".

Reply:
`Csz` for such autodetection is wrong, since, in theory, it can autodetect also the strings without the zero-terminator. I will stick with the first one.